designer/client/src/components/graph/node-modal/fragment-input-definition/settings/variants/fields/UserDefinedListInput.tsx

```diff
@@ -187,7 +189,7 @@ export const UserDefinedListInput = ({
                         validationLabelInfo={
                             temporaryValuesTyping && <CircularProgress size={"1rem"} sx={(theme) => ({ marginTop: theme.spacing(0.5) })} />
                         }
-                        expressionObj={{ language: ExpressionLang.SpEL, expression: temporaryListItem }}
+                        expressionObj={{ language: ExpressionLang.SpEL, expression: temporaryListItem.trim() }}
```

Comment:
What about backend? :) As I understand there is still the possibility to add duplicates?

Reply:
i wrote some additional tests  and it turned out that thie feature is only validated on FE so there is no nothing to fix  in BE. Backend validations could have been added but rather in other MR/task.